android/src/main/java/com/brentvatne/react/VideoManagerModule.kt

```diff
@@ -72,7 +72,7 @@ class VideoManagerModule(reactContext: ReactApplicationContext?) : ReactContextB
     @ReactMethod
     fun setFullScreen(fullScreen: Boolean, reactTag: Int) {
         performOnPlayerView(reactTag) {
-            it?.setFullscreen(fullScreen)
+            it?.handleFullScreenMode(fullScreen)
```

Comment:
merge issue here ?

Reply:
We need to use the `handleFullScreenMode` function because we must implement the logic `exoFullScreen.performClick()` to handle the fullscreen button correctly. As mentioned earlier, Media3 manages the fullscreen icon internally.

Reply:
Additionally, we can remove the fullscreen argument because the user doesn't handle it, Media3 manages the fullscreen status.

Reply:
be carefull, user can you fullscreen without controls !

Reply:
I've tested this way and everything works correctly.